api/TextTrack.json

```diff
@@ -150,7 +150,11 @@
         "__compat": {
           "description": "<code>cuechange</code> event",
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/TextTrack/cuechange_event",
-          "spec_url": "https://html.spec.whatwg.org/multipage/media.html#event-media-cuechange",
+          "spec_url": [
+            "https://html.spec.whatwg.org/multipage/media.html#event-media-cuechange",
+            "https://html.spec.whatwg.org/multipage/media.html#handler-texttrack-oncuechange",
+            "https://html.spec.whatwg.org/multipage/webappapis.html#handler-oncuechange"
```

Comment:
This link should not be included here, it's for "elements, Document objects, and Window objects".